### ISSUE TITLE
My valid loss =0, is this normal?

### ISSUE THREAD

**Issue by hdmjdp** [facebookresearch/denoiser#145]
[2023-03-02 10:04:54,240][denoiser.solver][INFO] - Fine tuning from pre-trained model dns64
[2023-03-02 10:04:54,621][denoiser.solver][INFO] - ----------------------------------------------------------------------
[2023-03-02 10:04:54,622][denoiser.solver][INFO] - Training...
[2023-03-02 10:20:06,015][denoiser.solver][INFO] - Train | Epoch 1 | 824/2063 | 0.9 it/sec | Loss 0.00540
[2023-03-02 10:27:25,421][denoiser.solver][INFO] - Train | Epoch 1 | 1236/2063 | 0.9 it/sec | Loss 0.00527
[2023-03-02 10:34:45,114][denoiser.solver][INFO] - Train | Epoch 1 | 1648/2063 | 0.9 it/sec | Loss 0.00523
[2023-03-02 10:42:03,012][denoiser.solver][INFO] - Train | Epoch 1 | 2060/2063 | 0.9 it/sec | Loss 0.00519
[2023-03-02 10:42:05,891][denoiser.solver][INFO] - Train Summary | End of Epoch 1 | Time 2231.27s | Train Loss 0.00519
[2023-03-02 10:42:05,892][denoiser.solver][INFO] - New best valid loss 0.0000
[2023-03-02 10:42:06,029][denoiser.solver][INFO] - ----------------------------------------------------------------------
[2023-03-02 10:42:06,029][denoiser.solver][INFO] - Overall Summary | Epoch 1 | Train 0.00519 | Valid 0.00000 | Best 0.00000

 I finnetuned dns64 on my own dataset, but the valid loss=0.

**Comment by adefossez** [comment]
It looks like your validation set is empty.

**Comment by hdmjdp** [comment]
> It looks like your validation set is empty.

And "Train Loss 0.00519"， is normal？ I got the pesq is 1.3.
